src/state/ducks/Wallet.ts

```diff
@@ -87,27 +87,26 @@ class Store {
     const identities: { [key: string]: IMyIdentity } = {}
 
     await Promise.all(
```

Comment:
Since the callback function is not async anymore, we would not need `Promise.all` anymore